test/source/mock/backend/backend-data.ts

```diff
@@ -7,9 +7,35 @@ import { OauthMock } from '../lib/oauth';
 // tslint:disable:no-null-keyword
 // tslint:disable:oneliner-object-literal
 
+export const keyManagerAutogenRules = {
+  "flags": [
+    "NO_PRV_BACKUP",
+    "ENFORCE_ATTESTER_SUBMIT",
+    "PRV_AUTOIMPORT_OR_AUTOGEN",
+    "PASS_PHRASE_QUIET_AUTOGEN",
+    "DEFAULT_REMEMBER_PASS_PHRASE",
+  ],
+  "key_manager_url": "https://localhost:8001/flowcrypt-email-key-manager",
+  "enforce_keygen_algo": "rsa2048",
+  "disallow_attester_search_for_domains": []
+};
+
+export type ClientConfiguration = {
+  // todo: should we somehow import the type from `client-configuration.ts`?
```

Comment:
`client-configuration.ts` is not under `core`, so it's not reachable from tests...
Should we do something about it?

Reply:
i'm not sure. will leave up to you